extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -566,4 +571,45 @@ private function getWikiaStatsData() {
 
 		return $wikiDetails['stats'];
 	}
+
+	private function getOtherLanguages( Title $title ) {
+		global $wgEnableLillyExt;
+
+		if ( empty( $wgEnableLillyExt ) ) {
+			return null;
+		}
+
+		$url = $title->getFullURL();
+		//$url = str_replace( '.rychu.wikia-dev.com', '.wikia.com', $url );
```

Comment:
Prototype-quality. (Lilly is a prototype as long as we stick to Star Wars communities): easy to test on one devbox without having the whole logic for handling different environments.